### DISCUSSION TITLE
Generates the input. Json file for nitrogen molecules

### DISCUSSION THREAD

**Discussion by 2329611140** [deepmodeling/deepmd-kit#1221] [Q&A]
This is the first time FOR me to come into contact with DEEPMD-Kit. At present, I am using deepMD-Kit on the MobaXterm server. I am looking for articles on building molecular models and training tests on the Internet. JSON file to use it?

**Comment by njzjz** [comment]
The manual is available at https://docs.deepmodeling.org/projects/deepmd/.

**Comment by AnguseZhang** [comment]
Please use English instead of Chinese to communicate, since we have many users who can't read Chinese. So this issue is declined.

**Reply by 2329611140** [reply]
I tried to modify the parameters in the input.json file of se_e2_A in the water folder on GitHub, and then used deepMD to collect training data of nitrogen molecules, but I received an error when running the code. The error code is as follows:
WARNING:tensorflow:From /home/feng/.local/conda/envs/deepmd/lib/python3.9/site-p_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated an
Instructions for updating:
non-resource variables are not supported in the long term
WARNING:root:Environment variable KMP_BLOCKTIME is empty. Use the default value
WARNING:root:Environment variable KMP_AFFINITY is empty. Use the default value g
WARNING:deepmd.train.run_options:Switch to serial execution due to lack of horov
Traceback (most recent call last):
  File "/home/feng/.local/conda/envs/deepmd/bin/dp", line 10, in <module>
    sys.exit(main())
  File "/home/feng/.local/conda/envs/deepmd/lib/python3.9/site-packages/deepmd/e
    train_dp(**dict_args)
  File "/home/feng/.local/conda/envs/deepmd/lib/python3.9/site-packages/deepmd/e
    jdata = j_loader(INPUT)
  File "/home/feng/.local/conda/envs/deepmd/lib/python3.9/site-packages/deepmd/c
    return json.load(fp)
  File "/home/feng/.local/conda/envs/deepmd/lib/python3.9/json/__init__.py", lin
    return loads(fp.read(),
  File "/home/feng/.local/conda/envs/deepmd/lib/python3.9/json/__init__.py", lin
    return _default_decoder.decode(s)
  File "/home/feng/.local/conda/envs/deepmd/lib/python3.9/json/decoder.py", line
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/home/feng/.local/conda/envs/deepmd/lib/python3.9/json/decoder.py", line
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 49 column 3 (char 1325)

The code of my JSON file is as follows. I hope you can help me to see what needs to be improved. Thank you very much！
{
    "_comment": " model parameters",
    "model": {
        "type_map":     ["N"],  
     "_comment": " type_map 应该和训练集中的顺序保持一致",
        "descriptor" :{
            "type":             "se_e2_a",
            "sel":              "auto",
            "rcut_smth":        0.50,
            "rcut":             5.00,
            "neuron":           [25, 50, 100],
            "resnet_dt":        false,
            "axis_neuron":      16,
            "seed":             1,
            "_comment":         " that's all"
        },
        "fitting_net" : {
            "neuron":           [240, 240, 240],
            "resnet_dt":        true,
            "seed":             1,
            "_comment":         " that's all"
        },
        "_comment":     " that's all"
    },

    "learning_rate" :{
        "type":         "exp",
        "decay_steps":  5000,
        "start_lr":     0.001,
        "stop_lr":      3.51e-8,
        "_comment":     "that's all"
    },

    "loss" :{
        "type":         "ener",
        "start_pref_e": 0.02,
        "limit_pref_e": 1,
        "start_pref_f": 1000,
        "limit_pref_f": 1,
        "start_pref_v": 0.02,
        "limit_pref_v": 1,
        "_comment":     " that's all"
    },

    "training" : {
        "training_data": {
            "systems":   [
"../data/N2",
		],
            "batch_size":       "auto",
            "_comment":         "that's all"
        },
        "numb_steps":   1000000,
        "seed":         10,
        "disp_file":    "lcurve.out",
        "disp_freq":    100,
        "save_freq":    1000,
        "_comment":     "that's all"
    },

    "_comment":         "that's all"
}

**Reply by wanghan-iapcm** [reply]
The error message is quite clear:
```
json.decoder.JSONDecodeError: Expecting value: line 49 column 3 (char 1325)
```
